commands/common/functions.py

```diff
@@ -0,0 +1,53 @@
+# Imported Modules
+from common.database.models import Users, Servers, ServerToUsers
+from sqlalchemy.orm import Session
+
+
+# USER ENTITY
+def returnUserEntity(discord_id: str, session: Session):
+    # queries user entity
+    user = session.query(Users).filter_by(discord_id=discord_id).first()
+    
+    if not user:
+        # entity doesn't exist; create new entity
+        user = Users(discord_id=discord_id)
+        session.add(user)
+        session.commit()
+        print(f"User ID ({discord_id}) added to the database")
```

Comment:
Can we start using logging for this? Such as:
`log.info(f'[INFO]: User ID ({discord_id}) added to the database')`
And for warning you can use .warn or errors with .error

Reply:
updated